src/predicate_lang/predicate_lang.ml

```diff
@@ -20,7 +20,7 @@ let union a = Union a
 
 let not_union a = compl (union a)
 
-let any = not_union []
+let any = Compl (Union [])
```

Comment:
Could be `Compl empty` if moved below. Also: I'm wondering if this deserves a separate constructor.

Reply:
(Also, could be `Inter []` for symmetry?)